docs/standard/glossary.md

```diff
@@ -213,7 +218,7 @@ Compilation happens on the developer's machine similar to the way a C++ compiler
 
 UWP was the first application framework supported by .NET Native. Now, we support building native console apps for Windows, macOS, and Linux.
```

Comment:
I think UWP is the only place it's currently used/supported.

Reply:
```suggestion
UWP is the application framework supported by .NET Native.
```